app/src/ui/lib/errors.tsx

```diff
@@ -4,13 +4,14 @@ import * as classNames from 'classnames'
 interface IErrorsProps {
   /** The class name for the internal element. */
   readonly className?: string
-
-  /** The children to be displayed as an error message. */
-  readonly children?: ReadonlyArray<JSX.Element>
 }
 
-/** An Errors element with app-standard styles. */
-export class Errors extends React.Component<IErrorsProps, void> {
+/**
+ * An Errors element with app-standard styles.
+ *
+ * Provide `children` elements to render as the content for the error element.
```

Comment:
You could create a different component interface (in name only) that implies that you should provide children. In my example, `ComposableComponent` would imply that you should give this component children.